src/Kernels/Receiver.cpp

```diff
@@ -2,7 +2,8 @@
  * @file
```

Comment:
Did you apply clang-format to this file? Then please also add it to the allowlist in `.ci/format.sh`

Reply:
I belive so, yes. But I added both the .h and .cpp to the format.sh